modules/siddhi-query-compiler/src/main/antlr4/org/wso2/siddhi/query/compiler/SiddhiQL.g4

```diff
@@ -27,14 +27,13 @@ parse
     ;
 
 error
-    : UNEXPECTED_CHAR 
+    : UNEXPECTED_CHAR
   //      {throw new SiddhiParserException("You have an error in your SiddhiQL at line " + $UNEXPECTED_CHAR.line + ", unexpected charecter '"+ $UNEXPECTED_CHAR.text +"'");}
     ;
 
 siddhi_app
     : (app_annotation|error)*
-      ( (definition_stream|definition_table|definition_trigger|definition_function|definition_window|definition_aggregation|error) (';' (definition_stream|definition_table|definition_trigger|definition_function|definition_window|definition_aggregation|error))* ';'?
-      || (execution_element|error) (';' (execution_element|error))* ';'?
+      ((execution_element|error) (';' (execution_element|error))* ';'?
       || (definition_stream|definition_table|definition_trigger|definition_function|definition_window|definition_aggregation|error) (';' (definition_stream|definition_table|definition_trigger|definition_function|definition_window|definition_aggregation|error))* (';' (execution_element|error))* ';'? )
```

Comment:
Can you explain what was wrong?

Reply:
In line 36(previous) and line 37(new) same thing was repeated. So antlr couldn't identify syntax errors in intellij idea because of this duplicate lines.

Reply:
I think we can get rid of ((execution_element|error) (';' (execution_element|error))* ';'? as there cannot exist a siddhi app without any definitions but only with queries.